c10/util/StringUtil.h

```diff
@@ -61,6 +61,19 @@ inline std::string Join(const std::string& delimiter, const Container& v) {
   return s.str();
 }
 
+
+inline std::vector<std::string> Split(const std::string& input, const std::string& delimiter) {
```

Comment:
There seems to be many different implementations of `Split` sprinkled all over the place. I'm not sure if we might want to consider unifying/merging them. Neither am I sure if this work needs to be a part of this PR.

Reply:
gut